config/rbac/rbac_role.yaml

```diff
@@ -89,7 +89,6 @@ rules:
   - machine.openshift.io
   resources:
   - machines
-  - machines/status
```

Comment:
This needs to go back

Reply:
> This needs to go back

Can you elaborate on the why? TIA.

Reply:
Since aws actuator is updating machine status. And, this PR is about removing fields, not about changing RBAC rules.

Reply:
done!